### PR TITLE
[env-man] Fixes for distance revision

### PR DESCRIPTION
**Pull request by cpollok**
Fixes the following bugs:
- Sealing action to distance 0.0 didn't work with goals. (Was closed to just 0.001 because of trying to avoid overshooting the limits.)
- Manpiulation of environment with distance being the same as the current joint-state raised an error. (No trajectory is calculated because the manipulation distance is too short. Just had to make sure there is no operation done on the nil result.)
- Rounding of angles before setting joint-state in event-handler just rounded up. That lead to problems when approaching the upper limit of joints.

### CODE REVIEW

File: cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp [hunk 1]
```diff
@@ -146,6 +146,7 @@ If there is no other method with 1 as qualifier, this method will be executed al
 
 
 (defun move-joint-by-event (event open-or-close)
+  (print event)
```
Comment (gaya-) [inline cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp]: What's that ;)

File: cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp [hunk 2]
```diff
@@ -158,16 +159,22 @@ If there is no other method with 1 as qualifier, this method will be executed al
...
+               new-joint-angle
+               0.0001)
+              10000)))
     (btr:set-robot-state-from-joints
      `((,joint-name
         ,new-joint-angle-rounded))
      object)))
 
 (defmethod cram-occasions-events:on-event open-container ((event cpoe:container-opening-event))
+  (print event)
```
Comment (gaya-) [inline cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp]: And that.

File: cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp (hunk 1 above)
Comment (cpollok) [reply cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp]: Whoops. Fixed.

File: cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp (hunk 2 above)
Comment (cpollok) [reply cram_3d_world/cram_bullet_reasoning_belief_state/src/event-handlers.lisp]: Whoopsie. Fixed.

**Pull request merged**